cbor.js

```diff
@@ -111,42 +271,21 @@ function encode(value) {
 
     switch (typeof value) {
       case "number":
+          if (isNaN(value))
```

Comment:
nans (and infinities) can be quickly encoded as float 16s under the assumption that the extended bits of NaNs are inaccessible - this might need to be removed or extended to cover infinities